debian/control

```diff
@@ -2,7 +2,7 @@ Source: mavlink-router
 Section: unknown
 Priority: optional
 Maintainer: Helge Bahmann <helge@auterion.com>
-Build-Depends: debhelper (>= 11)
+Build-Depends-Indep: debhelper (>= 11), pkg-config, libgtest-dev, systemd, python3-pip
```

Comment:
Hmmm I think `libgtest-dev` and `systemd` should be in `Build-Depends`, because their arch will vary depending on the arch of mavlink-router

Reply:
Yeah I was actually not sure about this. I will make this change